.travis.yml

```diff
@@ -18,20 +19,23 @@ install:
   - cd iverilog && autoconf && ./configure && make && sudo make install && cd ..
 
   # Install conda
-  - wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
+  - wget http://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh -O miniconda.sh
   - bash miniconda.sh -b -p $HOME/miniconda
   - export PATH=$HOME/miniconda/bin:$PATH
   - conda update --yes conda
-  - conda install --yes pytest numpy psutil qtpy pyqt pyyaml pyzmq pytables
-  - pip install cocotb
+  - conda create --yes -n test-environment python=$TRAVIS_PYTHON_VERSION pytest numpy psutil qtpy pyqt pyyaml pyzmq pytables
+  - source activate test-environment
+  - pip install cocotb==1.0.dev3
   - pip install xvfbwrapper  # fake x server for Qt gui tests
 
   # Install basil
-  - git clone -b v2.4.12 https://github.com/SiLab-Bonn/basil; cd basil; python setup.py develop; cd ..;
+  - git clone -b v3.0.0 https://github.com/SiLab-Bonn/basil; cd basil; python setup.py develop; cd ..;
```

Comment:
We should be able to pool just from pip now.

Reply:
Yes, basil-daq 3.0.0 is available with pip.